share/src/main/resources/alfresco/messages/slingshot_pt_BR.properties

```diff
@@ -549,17 +549,17 @@ menu.selected-items.manage-permissions=Gerir Permiss\u00f5es...
 menu.selected-items.deselect-all=Desmarcar tudo
 menu.selected-items.cloudSync=Syncronizar para nuvem
 menu.selected-items.cloudUnsync=Dessincronizar da nuvem
-menu.selected-items.cloudRequestSync=Pedir Sincroniza\u00e7\u00e3o
+menu.selected-items.cloudRequestSync=Solicitar sincroniza\u00e7\u00e3o
 
 ## Document Library Actions
-actions.document.assign-workflow=Iniciar o fluxo de trabalho
+actions.document.assign-workflow=Iniciar fluxo de trabalho
 actions.document.cancel-editing=Cancelar a edi\u00e7\u00e3o
 actions.document.cancel-editing-unlock=Desbloquear documento
 actions.document.change-type=Alterar tipo
 actions.document.copy-to=Copiar para...
 actions.document.delete=Excluir documento
-actions.document.download=Baixar
-actions.document.download-again=Baixar
+actions.document.download=Download
+actions.document.download-again=Download
 actions.document.download-original=Baixar original
```

Comment:
If you are going to replace the Baixar text with Download, then the line actions.document.download-original=Baixar original should be actions.document.download-original=Download do original